Automation/Redis_Cache/template.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Redis_nsc_redis_dev_usw2_thursday_name": {
+            "defaultValue": "nsc-redis-dev-usw2-thursday2",
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Cache/Redis",
+            "apiVersion": "2019-07-01",
+            "name": "[parameters('Redis_nsc_redis_dev_usw2_thursday_name')]",
+            "location": "West US 2",
+            "properties": {
+                "sku": {
```

Comment:
Not sure how standard these properties are, but you could read them in with a hash table so the user has more control over the redis cache they are creating if you wanted to.